src/hammer-vlsi/hammer_vlsi.py

```diff
@@ -1549,14 +1574,21 @@ def append_mmmc(cmd: str) -> None:
             list=library_set_name
         ))
         # extra junk: -opcond ...
+        rc_corner_name = "rc_cond"
+        append_mmmc("create_rc_corner -name {name} -temperature {temp} -qrc_tech {qrc}".format(
+            name=rc_corner_name,
+            temp=120, # TODO: this should come from tech config
+            qrc=self.get_qrc_tech()
```

Comment:
If there are no qrc techs, shouldn't we avoid emitting an empty "-qrc_tech" parameter?